### DISCUSSION TITLE
Image on top of Image

### DISCUSSION THREAD

**Discussion by nruim** [twbs/bootstrap#37876] [Q&A] [answered]
I am trying to place an image on top of another image.
![bronze](https://user-images.githubusercontent.com/13618038/212427416-448e76ad-df37-4cfb-95f4-3bda8509b617.png) - background

![lebron_james](https://user-images.githubusercontent.com/13618038/212427547-a760a21d-6ead-43af-b8af-f2ca3df227bf.png) - front image

to make

![bronze_james](https://user-images.githubusercontent.com/13618038/212427811-bdc55f3d-a7c8-459a-b0d5-ebe7dd762e2a.png)

The photos used are just for example.

Any help would be greatly appreciated. Thank you!

**Comment by JosephShown** [comment, marked as answer]
Thank you for your question. First you need an html element to hold the background image then you give it the style background-image. Within that element you then put an img element with your foreground image element. Here is a code example below with your exact pictures as an example. You can test this on any code editor such as [https://www.w3schools.com/tryit/tryit.asp?filename=tryhtml_hello] just copy and paste the below code and hit run or whatever applicable button. Keep in mind you may have to do some additional work to size and position images properly, as i did with the width setting below.
```
<div style="width: 300px; background-image: url(https://user-images.githubusercontent.com/13618038/212427416-448e76ad-df37-4cfb-95f4-3bda8509b617.png)">
<img src="https://user-images.githubusercontent.com/13618038/212427547-a760a21d-6ead-43af-b8af-f2ca3df227bf.png" />
<div>
```
